### DISCUSSION TITLE
账户收不到授权码，无法完成专业版授权。

### DISCUSSION THREAD

**Discussion by JackieIris** [getlantern/lantern-forum-cn#644] [General]
我的邮箱账号是：liurain0312@gmail.com; 使用邮箱授权专业版无法收到验证码。支付订单找不到了，买了好几年的，信用卡记录只能查一年的。
以前都可以收到授权码，这次试了多少次了，都不行，等了几天了都没有收到，尝试了好几次都不行，一年马上就能收到（Mac, Android, window虚拟机都试了，不行）。原来不知道还可以在这里提问，给官网的邮箱发了不知道多少封邮件了，就是没回复。
蓝灯我用了快十年了，不是最好的，但你习惯了就知道什么时候好用，我有好几个VPN，蓝灯主要用来下载文件比较快，平时用其他VPN查网页。所以请不要辜负我的信任。
还有，就是每次通过应用反馈都没有得到过反馈，而且默认的反馈邮箱marktwain2017fb@gmail.com是我第一次购买的邮箱，后来已经改成我现在的liurain0312@gmail.com,最开始能联系上的时候就反馈过，到现在也没有改过来。
已经三个多月了，希望能尽快解决！！！

**Comment by MaxMKMax** [comment]
@JackieIris 您好. 我帮您提交问题上去了. 我们会尽快给您答复的. 谢谢您的等待与支持.

**Comment by MaxMKMax** [comment]
@JackieIris 
您好. 这是您专属的激活码. 谢谢您对我们的支持!
4RK4M-Y268F-9432G-HCX7K-DBJHX

**Comment by JackieIris** [comment]
万分感谢得到您的回复。以我的理解，激活码只能在这里使用，可是您给的激活版并不能成功激活，如图所示。
<img width="692" alt="authorization failure" src="https://github.com/getlantern/lantern-forum-cn/assets/25077102/968110a2-ed67-4573-b54b-4e71c4026f84">

**Reply by JackieIris** [reply]
@Mike777Khoo  请即时查看并处理我的回复。在这里回复激活码不是谁都可以看到？别人拿着这个激活码不是也能激活？所以麻烦核实一下是否有人拿着这个激活码在用，可以禁用他MAC地址的使用。
处理了盗用的问题，如果有新的激活码或处理办法请直接发到我的邮箱。
谢谢，期待您的回复！

**Reply by JackieIris** [reply]
@Mike777Khoo 我等了一周了，也没有得到您的回复。十年用户的耐心就这样一点一点被消耗掉。希望可以尽快解决。
如果无法解决，烦请退款。
谢谢！

**Comment by JackieIris** [comment]
@Mike777Khoo 今天终于激活成功了，感谢。iOS的专业版授权希望可以尽快解决，这个问题其实已经等待好几年的时间了。
